src/System.Net.Requests/tests/RequestStreamTest.cs

```diff
@@ -11,11 +11,18 @@
 
 namespace System.Net.Tests
 {
-    [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Tests hang on .NET Framework")]
     public class RequestStreamTest
     {
         readonly byte[] buffer = new byte[1];
 
+        static RequestStreamTest()
+        {
+            // On .NET Framework, the default limit for connections/server is very low (2).
+            // On .NET Core, the default limit is higher. Since these tests run in parallel,
+            // the limit needs to be increased to avoid timeouts when running the tests.
+            ServicePointManager.DefaultConnectionLimit = int.MaxValue;
```

Comment:
I don't think this is about a leak.  It is more about parallel requests especially true given how Xunit works in these libraries.  Since everything is using either the Azure echo server or the loopback server, that means a single ServicePoint object is being created and used.  And therefore, there will be a limit on how many connections can be open (in parallel) at once.